tests/nlu/utilities.py

```diff
@@ -11,6 +11,14 @@ def write_file_config(file_config):
         return f
 
 
+# check if os sequences e sentences is loaded correctly
+def get_feature_vectors(sequence, sentence):
```

Comment:
Please add type annotations for the arguments and return types. Also, our convention is to put descriptions into the function's doc string, i.e. 
```
def get_feature_vectors(sequence: ???, sentence: ???) -> Tuple[Optional[???], Optional[???]]:
  """Retrieves feature vectors from the features.
  
  Args:
    sequence: Features of individual tokens.
    sentence: Features of the entire string.
  
  Returns:
    Feature vectors (or None) for sequence and sentence features.
  """
  ...
```
You'll have to substitute the right thing for the question marks (just follow the return type of `test_message1.get_sparse_features`). Sorry I didn't see this earlier.